app/src/main/java/ru/androidacademy/gooseeye/data/Repository.kt

```diff
@@ -0,0 +1,108 @@
+package ru.androidacademy.gooseeye.data
+
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+
+class Repository() {
+
+    private val jsonFormat = Json { ignoreUnknownKeys = true }
+
+    @Serializable
+    private class JsonGenre(val id: Int, val name: String)
+
+    @Serializable
+    private class JsonActor(
+        val id: Int,
+        val name: String,
```

Comment:
i recommend you to add `@ SerialName` for all fields. 
You can avoid mistakes like when you rename fields. And I'm also not sure about how obfuscation affects kotlin serialization.